include/rviz_visual_tools/imarker_simple.hpp

```diff
@@ -39,31 +39,28 @@
 #pragma once
```

Comment:
The deprecation hpp, above, could just be deleted since C++14 has [[deprecated]]